src/Ulid/Ulid.cs

```diff
@@ -459,7 +463,6 @@ public override string ToString()
 #endif
         }
 
-#if NET6_0_OR_GREATER
```

Comment:
Interface declarations are only possible with net6 or greater, but I thought we could add implementations, so removed some of extra `#if`.   Any problem ..?